src/Riprap/Riprap.php

```diff
@@ -60,5 +60,44 @@ public function getEvents($resource_id) {
        drupal_set_message(t("Sorry, there has been an error connecting to Riprap, please refer to the system log"), 'error');
     }
   }
+  public function getFailures($resource_id) {
```

Comment:
Sorry @ajstanley, I can't figure out why you created this new method and why you replaced `getEvents()` in IslandoraRiprapController.php with it. Can you explain?

Reply:
OK, thanks. Should you remove the entire `getFailures()` method? Grepping through the module code only shows the method definition, it's not used anywhere. Sorry if I'm being dense, I've only had two cups of joe this morning :smile: